data/json/professions.json

```diff
@@ -661,7 +661,7 @@
     "type": "profession",
     "ident": "basketball_player",
     "name": "Basketball Player",
-    "description": "It was going to be your first major game, but then the Cataclysm struck.  Thanks to your quick feet, you were among the lucky few to survive and escape from the creatures.",
+    "description": "Your first major game was abruptly cancelled when the Cataclysm struck in the third quarter.  Quick feet and good reflexes meant you were among the lucky few to escape the stadium alive.",
```

Comment:
The cataclysm wasn't too much of a sudden event (like say, nukes falling in fallout), I think it'd make more sense for the game to be cancelled in advance.

Reply:
Maybe "cancelled when zombies stormed into the stadium"?